test/controllers/managing.officer.corporate.controller.spec.ts

```diff
@@ -0,0 +1,30 @@
+jest.mock("ioredis");
+jest.mock('../../src/controllers/authentication.controller');
+
+import { describe, expect, test, jest } from '@jest/globals';
+import { NextFunction, Request, Response } from "express";
+import request from "supertest";
+import app from "../../src/app";
+import { authentication } from "../../src/controllers";
+import { MANAGING_OFFICER_CORPORATE_URL } from "../../src/config";
+
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const MANAGING_OFFICER_CORPORATE_PAGE_TITLE = "Tell us about the corporate managing officer";
```

Comment:
page heading should go in https://github.com/companieshouse/overseas-entities-web/blob/main/test/__mocks__/text.mock.ts with the others